### PR TITLE
Add links to TBS Dependencies to Buildpack release notes

### PR DESCRIPTION
**Pull request by kcoriordan**
This will be cherry-picked to 1.4.0 and 1.4.1
This relates to this pivitol issue: https://www.pivotaltracker.com/n/projects/2537714/stories/184252581
and this Jira issue: https://jira.eng.vmware.com/browse/TAAP-743

### CODE REVIEW

File: tanzu-build-service/dependencies.hbs.md [hunk 1]
```diff
@@ -164,22 +164,22 @@ The `full` dependencies contain the following buildpacks in Tanzu Application Pl
...
-| Node.js Buildpack for VMware Tanzu | 1.17.1 | Bionic, Jammy |
-| Python Buildpack for VMware Tanzu | 2.3.1 | Bionic, Jammy |
-| Ruby Buildpack for VMware Tanzu | 1.5.0 | Bionic, Jammy |
-| Go Buildpack for VMware Tanzu | 2.0.4 | Bionic, Jammy |
+| Java Buildpack for VMware Tanzu | [8.0.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-release-notes.html#tanzu-java-buildpack-8.0.0) | Bionic, Jammy |
+| Java Native Image Buildpack for Tanzu | [6.39.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-native-image-release-notes.html#tanzu-java-native-image-buildpack-6.39.0) | Bionic, Jammy |
+| .NET Core Buildpack for VMware Tanzu | [1.19.2](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-dotnet-core-release-notes.html#tanzu.net-core-buildpack-1.19.2-full) | Bionic, Jammy |
+| Node.js Buildpack for VMware Tanzu | [1.17.1](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-nodejs-release-notes.html#tanzu-node.js-buildpack-1.17.0-full) | Bionic, Jammy |
```
Comment (kcoriordan) [inline tanzu-build-service/dependencies.hbs.md:170]: I don't see a version 1.17.1 - only 1.17.**0**
?

File: tanzu-build-service/dependencies.hbs.md [hunk 2]
```diff
@@ -164,22 +164,22 @@ The `full` dependencies contain the following buildpacks in Tanzu Application Pl
...
+| Python Buildpack for VMware Tanzu | [2.3.1](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-python-release-notes.html#tanzu-python-buildpack-2.3.1-full) | Bionic, Jammy |
+| Ruby Buildpack for VMware Tanzu | [1.5.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-ruby-release-notes.html#tanzu-ruby-buildpack-1.5.0-full) | Bionic, Jammy |
+| Go Buildpack for VMware Tanzu | [2.0.4](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-go-release-notes.html#tanzu-go-buildpack-2.0.4-full) | Bionic, Jammy |
 | PHP Buildpack for VMware Tanzu | 2.0.0 | Bionic, Jammy |
```
Comment (kcoriordan) [inline tanzu-build-service/dependencies.hbs.md:174]: I can't find the release note entry for this?

**Comment by mgibson1121** [comment]
This should be just for 1.4.  The buildpack versions will change for 1.4.1 and we'll have to update.  Each release of TAP (either a patch or a minor comes out each month) we ship new buildpack versions in TAP.

File: tanzu-build-service/dependencies.hbs.md [hunk 3]
```diff
@@ -118,17 +118,17 @@ The `lite` dependencies contain the following buildpacks in Tanzu Application Pl
...
-| Node.js Buildpack for VMware Tanzu (Lite) | 1.17.1 | Bionic, Jammy |
-| Python Buildpack for VMware Tanzu (Lite) | 2.3.1 | Bionic, Jammy |
-| Go Buildpack for VMware Tanzu (Lite) | 2.0.4 | Bionic, Jammy |
-| Web Servers Buildpack for VMware Tanzu (Lite) | 0.4.1 | Bionic, Jammy |
-| Ruby Buildpack for VMware Tanzu (Lite) | 1.5.0 | Bionic, Jammy |
-| Procfile Buildpack for VMware Tanzu (Lite) | 5.4.0 | Bionic, Jammy |
-| Base Stack of Ubuntu Bionic for VMware Tanzu | 1.2.34 | Bionic |
-| Base Stack of Ubuntu Jammy for VMware Tanzu | 0.1.16 | Jammy |
+| Java Buildpack for VMware Tanzu (Lite) | [8.0.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-release-notes.html#tanzu-java-buildpack-8.0.0) | Bionic, Jammy |
+| Java Native Image Buildpack for Tanzu (Lite) | [6.39.0] | Bionic, Jammy |
```
Comment (mgibson1121) [inline tanzu-build-service/dependencies.hbs.md:122]: Here is the link for the Kava native image release https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-native-image-release-notes.html#tanzu-java-native-image-buildpack-6.39.0

**Comment by mgibson1121** [comment]
lgtm

File: tanzu-build-service/dependencies.hbs.md (hunk 2 above)
Comment (mgibson1121) [reply tanzu-build-service/dependencies.hbs.md:174]: I think PHP doesn't have one yet because it's so new.

**Comment by kcoriordan** [comment]
OK, I committed directly into branch 1.4. Closing this issue: https://jira.eng.vmware.com/browse/TAAP-743

**Pull request closed without merging**